@shared/api/external.ts

```diff
@@ -81,7 +81,7 @@ export const submitBlob = async (
   const { network, networkPassphrase, accountToSign } = opts;
   try {
     response = await sendMessageToContentScript({
-      transactionXDR: blob,
+      transactionXdr: blob,
```

Comment:
can we call this var `transactionBlob` for clarity?

You'll need to change the var name in `freighterApiMessageListener` as well

Reply:
I also noticed that the function in `freighterApiMessageListener` isn't using the `network` and  `networkPassphrase` params. Can we remove?

Reply:
ah yeah totally we can get rid of those, assuming that a blob has no relation to a specific network.

Reply:
oh but I guess it wouldn't be transactionBlob, the data can be any b64 encoded data so how about just blob?

Reply:
addressed these in 11e8d4cef1b23309fefbd14edc064323e1221e87